Add change log notes for removal of 4.1.1

venkatvemuri28 opened the pull request:
- Updates Appendix A. Change Log to reflect the November 10, 2019 entry (AA → A), now marked as superseded.
- Adds an April 7, 2023 entry noting the reversion to Level AA.
- Resolves #3468.

<img width="888" height="566" alt="Screenshot 2025-10-14 at 21 02 48" src="https://github.com/user-attachments/assets/c122f7bd-d7b7-4613-8e00-233040e5fbaf" />

LIVE VIEW : https://deploy-preview-4668--wcag2.netlify.app/guidelines/

mbgower commented:
We do not normally record contents that change before the FPWD or between the FPWD and the publishing of the Rec that are not incorporated as part of the published rec (my emphasis in bold):
> substantive changes **incorporated** into WCAG 2.2 since WCAG 2.1, as well as changes made to 2.2 since its original publication on 05 October 2023

Since the change in level of Focus Visible was reverted before publication date, I think either entry is necessary.

As an example, the proposed Focus Appearance (Minimum) that was originally brought in as level AA is not recorded as moving to a AAA; we only record the addition and the name change. There is no mention of Focus Appearance (Enhanced) which also appeared in the August 2020 draft but was later removed.

venkatvemuri28 commented:
> @venkatvemuri28 as mentioned in the issue, the 2.4.7 changes are not needed. Are you open to making this PR an update for 4.1.1's removal? Suggested text is:
> 
> > 2023-01-25: 4.1.1 Parsing marked as obsolete and removed.

Updated.

bruce-usab commented:
Discussed briefly on Backlog call and moved to Ready for approval.
